superbench/benchmarks/micro_benchmarks/memory_copy_bw_cuda_performance.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""Module of the Cuda memory performance benchmarks."""
+
+import os
+
+from superbench.common.utils import logger
+from superbench.benchmarks import BenchmarkRegistry, Platform
+from superbench.benchmarks.micro_benchmarks import MicroBenchmarkWithInvoke
+
+
+class MemBwCuda(MicroBenchmarkWithInvoke):
+    """The Cuda memory bus bandwidth performance benchmark class."""
+    def __init__(self, name, parameters=''):
+        """Constructor.
+
+        Args:
+            name (str): benchmark name.
+            parameters (str): benchmark parameters.
+        """
+        super().__init__(name, parameters)
+
+        self._bin_name = 'bandwidthTest'
+        self.__mem_types = ['htod', 'dtoh', 'dtod']
+
+    def add_parser_arguments(self):
```

Comment:
Why no arguments?

Reply:
I'm not sure about the arguments, I didn't see the following args in 1.0version cuda11.0 docker. I should ask Guoshuai.